frontend/src/routes/(authenticated)/project/[project_code]/ChangeMemberRoleModal.svelte

```diff
@@ -5,25 +5,26 @@
   import t from '$lib/i18n';
   import { z } from 'zod';
   import { _changeProjectMemberRole } from './+page';
+  import type { UUID } from 'crypto';
 
   export let projectId: string;
 
-  const schema = z.object({
-    role: z.enum([ProjectRole.Editor, ProjectRole.Manager]),
+  $: schema = z.object({
```

Comment:
why is this now a $ property?

Reply:
Leftover from when the component handled both project and org memberships and needed to change its schema based on the `roleType` prop. Removed in commit 739524f0.